Assets/MixedRealityToolkit/Utilities/Scripts/Tagalong.cs

```diff
@@ -292,10 +292,14 @@ private bool AdjustTagalongDistance(Vector3 cameraPosition, out Vector3 newPosit
                             {
                                 Light clonedLight = Instantiate(DebugPointLight, closestHit.point, Quaternion.identity) as Light;
                                 clonedLight.color = Color.red;
-                                DestroyObject(clonedLight, 1.0f);
-                            }
+#if UNITY_2018_2_OR_NEWER
+								Object.Destroy(clonedLight, 1.0f);
+#else
+								DestroyObject(clonedLight, 1.0f);
+#endif
+							}
 #if UNITY_EDITOR
-                            DebugDrawLine(DebugDrawLines, cameraPosition, targetCoord, Color.red);
+							DebugDrawLine(DebugDrawLines, cameraPosition, targetCoord, Color.red);
```

Comment:
Formatting seems broken in a few of these classes

Reply:
I'm not sure how to fix that, when opening the scripts in VS the code aligns just fine. And 2018.2 is still beta. People might still want to try the toolkit, there's no harm to it due to the directives and the code can be filtered by it when tackling 2018.2 for real.

Reply:
The misalignment is because the whitespace has changed from spaces to tabs.